README.md

```diff
@@ -642,6 +642,9 @@ Sage is a free open-source mathematics software system licensed under the GPL. I
   + [Crypto++](http://www.cryptopp.com/): is a free and open source C++ class library of cryptographic algorithms and schemes.
   + [HElib](https://github.com/shaih/HElib): an Implementation of homomorphic encryption.
   + [NaCl](http://nacl.cr.yp.to/): high-speed software library for network communication, encryption, decryption, signatures, etc.
++ Python
+  + [pycrypto](https://www.dlitz.net/software/pycrypto/): Widely used package containing various cryptographic modules and functions
+  + [Standard Library Cryptographic Services](https://docs.python.org/2/library/crypto.html): A set of utilities included with Python.
```

Comment:
You're right! Done.

On Mon, Jan 5, 2015 at 12:32 AM, Eric Mill <notifications@github.com> wrote:

> In README.md
> <https://github.com/MaciejCzyzewski/retter/pull/2#discussion-diff-22446739>
> :
>
> > @@ -642,6 +642,9 @@ Sage is a free open-source mathematics software system licensed under the GPL. I
> >    + [Crypto++](http://www.cryptopp.com/): is a free and open source C++ class library of cryptographic algorithms and schemes.
> >    + [HElib](https://github.com/shaih/HElib): an Implementation of homomorphic encryption.
> >    + [NaCl](http://nacl.cr.yp.to/): high-speed software library for network communication, encryption, decryption, signatures, etc.
> > ++ Python
> > +  + [pycrypto](https://www.dlitz.net/software/pycrypto/): Widely used package containing various cryptographic modules and functions
> > +  + [Standard Library Cryptographic Services](https://docs.python.org/2/library/crypto.html): A set of utilities included with Python.
>
> Probably worth distinguishing between Python 2 and Python 3 here,
> especially since Python 3 has finally gained some real steam.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MaciejCzyzewski/retter/pull/2/files#r22446739>.
>



-- 
Santiago Basulto.-